.travis.yml

```diff
@@ -2,9 +2,11 @@ before_script:
  - git fetch --unshallow # we need a full clone for tests
  - git config --global user.email "you@example.com"
  - git config --global user.name "Your Name"
+ - gem install bundler
```

Comment:
Yes. https://github.com/travis-ci/travis-ci/issues/5239

But tests failed. (Ummm)
I will try to solve this problem. Please wait a moment.